<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Beginning</class>
 <widget class="QDialog" name="Beginning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模式选择</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
border:none;
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
 stop:0 #ff9569,stop:1 #e92758);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="title">
    <string>模式选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="singlePlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>人机对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doublePlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>双人对战</string>
    </property>
   </widget>
   <widget class="QPushButton" name="networkPlay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>网络对战</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="AHU">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rule">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;}
QPushButton:hover{
color:yellow;
}</string>
   </property>
   <property name="text">
    <string>游戏规则</string>
   </property>
  </widget>
  <widget class="QPushButton" name="worker">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:none;}
QPushButton:hover{
color:yellow;
}</string>
   </property>
   <property name="text">
    <string>制作人员</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
